akka-http/src/main/scala/akka/http/javadsl/server/directives/RouteDirectives.scala

```diff
@@ -38,14 +37,30 @@ abstract class RouteDirectives extends RespondWithDirectives {
   /**
    * Java-specific call added so you can chain together multiple alternate routes using comma,
    * rather than having to explicitly call route1.orElse(route2).orElse(route3).
+   * @deprecated Use the `RouteDirectives.concat` method instead.
    */
+  @Deprecated
```

Comment:
We usually use the scala `@deprecated` which takes an explanation and a `since` version which makes it easy to see when the deprecation happened. Does that not end up in the Java code (I think someone claimed that elsewhere) or why are we using the Java annotation + javadoc-tag here instead?

The since "1.10.4" information would be useful regardless.

Reply:
Ah, I saw @jlprat s comment now, maybe ignore the part about the `since` then